src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/None.xaml

```diff
@@ -0,0 +1,95 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Yes i'll file a bug against CPS and in fact, I shouldn't be checking this in. It slipped in.